Presentation.Web/Controllers/API/V2/Internal/ItSystems/ItSystemLocalOptionTypesInternalV2Controller.cs

```diff
@@ -0,0 +1,115 @@
+using System.Net;
+using System.Web.Http;
+using Core.ApplicationServices.LocalOptions;
+using Core.DomainModel.ItSystem;
+using Core.DomainModel.LocalOptions;
+using Presentation.Web.Controllers.API.V2.Common.Mapping;
+using Presentation.Web.Infrastructure.Attributes;
+using Presentation.Web.Models.API.V2.Internal.Request.Options;
+using Presentation.Web.Models.API.V2.Internal.Response;
+using Swashbuckle.Swagger.Annotations;
+
+namespace Presentation.Web.Controllers.API.V2.Internal.ItSystems
+{
+    [RoutePrefix("api/v2/internal/it-systems/{organizationUuid}/local-option-types")]
+    public class ItSystemLocalOptionTypesInternalV2Controller: InternalApiV2Controller
+    {
+
+        private readonly IGenericLocalOptionsService<LocalBusinessType, ItSystem, BusinessType> _businessTypeService;
+        private readonly ILocalOptionTypeResponseMapper _responseMapper;
+        private readonly ILocalOptionTypeWriteModelMapper _writeModelMapper;
+
+        public ItSystemLocalOptionTypesInternalV2Controller(IGenericLocalOptionsService<LocalBusinessType, ItSystem, BusinessType> businessTypeService, ILocalOptionTypeResponseMapper responseMapper, ILocalOptionTypeWriteModelMapper writeModelMapper)
+        {
+            _businessTypeService = businessTypeService;
+            _responseMapper = responseMapper;
+            _writeModelMapper = writeModelMapper;
+        }
+
+        [HttpGet]
+        [Route("business-types")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(IEnumerable<LocalRegularOptionResponseDTO>))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        public IHttpActionResult GetLocalBusinessTypes([NonEmptyGuid][FromUri] Guid organizationUuid)
+        {
+            if (!ModelState.IsValid) return BadRequest();
+
+            return _businessTypeService.GetByOrganizationUuid(organizationUuid)
+                .Select(_responseMapper.ToLocalRegularOptionDTOs<ItSystem, BusinessType>)
+                .Match(Ok, FromOperationError);
+        }
+
+        [HttpGet]
+        [Route("business-types/{optionUuid}")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(LocalRegularOptionResponseDTO))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        public IHttpActionResult GetLocalBusinessTypeByOptionId([NonEmptyGuid][FromUri] Guid organizationUuid, [FromUri] Guid optionUuid)
+        {
+            if (!ModelState.IsValid) return BadRequest();
+
+            return _businessTypeService.GetByOrganizationUuidAndOptionUuid(organizationUuid, optionUuid)
+                .Select(_responseMapper.ToLocalRegularOptionDTO<ItSystem, BusinessType>)
+                .Match(Ok, FromOperationError);
+        }
+
+        [HttpPost]
+        [Route("business-types/")]
```

Comment:
Didn't we decide on the Choice type specific controllers? This Usage local option types will get quite crowded